### DISCUSSION TITLE
2024年愚人节版本被错误地分类

### DISCUSSION THREAD

**Discussion by githubusera2034324** [Meloong-Git/PCL#3719] [讨论]
2024年愚人节版本被分类到快照版中，强迫症真的很难受😅

**Comment by MoYuan-CN** [comment]
可以正常分类了的说
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/31eaa19b-7a1c-4918-9178-a298f58c6e54)

**Comment by Deep-Dark-Forest** [comment]
https://github.com/Hex-Dragon/PCL2/issues/3591

**Comment by wuliaodexiaoluo** [comment]
<details> 

<summary> Off Topic </summary>

~45 分钟刷了 22 条邮件~

~我不知道该说啥好~

![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/ec2775d1-36f2-46f5-bcb6-55ae946be515)

</details>

这不是已经更新了？
